lib/auth/middleware.go

```diff
@@ -343,6 +343,7 @@ func getCustomRate(endpoint string) *ratelimit.RateSet {
 	// Account recovery RPCs.
 	case
 		"/proto.AuthService/ChangeUserAuthentication",
+		"/proto.AuthService/ChangePassword",
```

Comment:
Are we happy w/ introducing Rate Limiting on this endpoint ? It seemed like the sensible thing to do.